### DISCUSSION TITLE
please add documentation for the animation , drag drop , and scroll animation

### DISCUSSION THREAD

**Discussion by bobwatcherx** [zauberzeug/nicegui#761] [Ideas / Feature Requests]
### Question

I want to use animation , drag drop and scroll animation effect . But couldn't find an example. And is there a way to do a full page scroll like tiktok. Thank You

**Comment by rodja** [comment]
This features are mainly html/css related. You should be able to adapt the wanted behaviour from examples you can find on the internet. For drag and drop we have made an example quite a while ago: https://github.com/zauberzeug/nicegui/tree/main/examples/trello_cards

**Comment by rodja** [comment]
But of course more examples are always nice. We welcome any contributions!
